docs/integrations/glue_metastore.md

```diff
@@ -0,0 +1,394 @@
+name
+mark
+EOF
+
+cat <<EOF | lakectl fs upload lakefs://$repo/$branch/$data_path/type=turtle/weight=12/partition.csv -s -
+name
+mrgreen
+EOF
+
+cat <<EOF | lakectl fs upload lakefs://$repo/$branch/$data_path/type=bear/weight=30/partition.csv -s -
+name
+winnie
+EOF
+
+lakectl commit lakefs://$repo/$branch -m "add table data"
+```
+The files added to lakeFS should look something like this: 
+
+![lakeFS Uploaded CSV Files]({{ site.baseurl }}/assets/img/csv_export_hooks_data.png)
+
+  </div>
+  <div markdown="1" id="spark-parquet">
+#### Spark Parquet Example
+
+The following example will create the table in spark DF, write to lakeFS via S3 Gateway and commit the data.
+
+```python
+import lakefs_client
+from lakefs_client.models import CommitCreation
+from lakefs_client.client import LakeFSClient
+from pyspark.sql import SparkSession
+# lakefs config 
+lakefs_endpoint='http://localhost:8000' 
+lakefs_access_key_id='<LAKEFS_KEY_ID<'
+lakefs_secret_access_key='<LAKEFS_SECRET_ACCESS_KEY>'
+# init lakeFS client 
+configuration = lakefs_client.Configuration(host=lakefs_endpoint,username=lakefs_access_key_id,password=lakefs_secret_access_key)
+configuration.verify_ssl = False
+lakefsApi = LakeFSClient(configuration)
+# init spark session 
+builder = SparkSession.builder.appName("MyApp") \
+    .config("spark.hadoop.fs.s3a.access.key", lakefs_access_key_id) \
+    .config("spark.hadoop.fs.s3a.secret.key",lakefs_secret_access_key) \
+    .config("spark.hadoop.fs.s3a.path.style.access", "true") \
+    .config("spark.hadoop.fs.s3a.endpoint", lakefs_endpoint) \
+    .config("spark.hadoop.fs.s3a.impl", "org.apache.hadoop.fs.s3a.S3AFileSystem")
+spark = builder.getOrCreate()
+# write table data as parquet files to lakeFS 
+repo="catalogs"
+ref = "main"
+tablePath = f"s3a://{repo}/{ref}/tables/animals"
+data=[
+    ("axolotl", 10, "jayson"),
+    ("axolotl", 10, "john"),
+    ("axolotl", 12, "mark"),
+    ("turtle", 12, "mrgreen"),
+    ("bear", 30, "winnie"),
+]
+columns=["type","weight","name"]
+df=spark.createDataFrame(data,columns)
+df.write.partitionBy("type","weight").mode("overwrite").parquet(f"{tablePath}/data.parquet")
+# commit data to lakeFS 
+lakefsApi.commits.commit(repository=repo,branch=ref,commit_creation=CommitCreation(message="data"))
+```
+  </div>
+
+</div>
+
+### Add Glue Exporter
+
+The current step requires 2 things: 
+1. Lua script.
+2. Action to trigger the script.
+
+#### Lua packages (Exporters)
+
+- [symlink_exporter]({% link howto/hooks/lua.md %}#lakefscatalogexportsymlink_exporter)
+- [glue_exporter]({% link howto/hooks/lua.md %}#lakefscatalogexportglue_exporter)
+
+#### Exporter Script 
+
+##### 1. Create Lua script:
+  
+For the simple strategy of creating a glue table per repo / branch / commit we can simply copy-paste the following script and re-use it.
+Upload the script to `scripts/animals_exporter.lua` (could be any path).
```

Comment:
```suggestion
### The exporter script

Upload the following script to your main branch under `scripts/animals_exporter.lua` (or a path of your choice).
```

Reply:
like it! but with a little twist I kept the libraries reference.